docs/misc/math-expr.md

```diff
@@ -177,14 +177,17 @@ See javadoc of java.lang.Math for detailed explanation for each function.
 | array_offset_of(arr,expr) | returns the 0 based index of the first occurrence of expr in the array, or `-1` or `null` if `druid.generic.useDefaultValueForNull=false`if no matching elements exist in the array. |
 | array_ordinal_of(arr,expr) | returns the 1 based index of the first occurrence of expr in the array, or `-1` or `null` if `druid.generic.useDefaultValueForNull=false` if no matching elements exist in the array. |
 | array_prepend(expr,arr) | adds expr to arr at the beginning, the resulting array type determined by the type of the array |
-| array_append(arr1,expr) | appends expr to arr, the resulting array type determined by the type of the first array |
+| array_append(arr,expr) | appends expr to arr, the resulting array type determined by the type of the first array |
```

Comment:
> the resulting array type determined by the type of the first array

Is this true? Should it mention how the type is determined per https://github.com/apache/druid/pull/11104/files#diff-7badc739fd6eef810cbd31950d52f28267273e129b9371beeea0bc5d125da7f7R267?

Reply:
Ah, it is true. The method you linked is used when converting the values from the input binding to expressions. Within expression evaluation the types are already decided, and so the first array type dictates the output type (see the case expressions in the array function eval methods)

Reply:
Ah cool, thanks for the explanation :+1: